vscode/vscode/vscode.ml

```diff
@@ -2140,9 +2254,26 @@ module Workspace = struct
     val onDidOpenTextDocument : OnDidOpenTextDocument.t
       [@@js.global "vscode.workspace.onDidOpenTextDocument"]
 
+    val onDidSaveTextDocument : OnDidSaveTextDocument.t
+      [@@js.global "vscode.workspace.onDidSaveTextDocument"]
+
     val onDidCloseTextDocument : OnDidCloseTextDocument.t
       [@@js.global "vscode.workspace.onDidCloseTextDocument"]
 
+    val onDidChangeTextDocument : OnDidChangeTextDocument.t
+      [@@js.global "vscode.workspace.onDidChangeTextDocument"]
+
+    val applyEdit : edit:WorkspaceEdit.t -> bool Promise.t
+      [@@js.global "vscode.workspace.applyEdit"]
+
+    val registerTextDocumentContentProvider :
+      scheme:string -> provider:TextDocumentContentProvider.t -> Disposable.t
+      [@@js.global "vscode.workspace.registerTextDocumentContentProvider"]
+
+    val asRelativePath :
+      pathOrUri:([ `String of string | `Uri of Uri.t ][@js.union]) -> string
+      [@@js.global "vscode.workspace.asRelativePath"]
```

Comment:
Missing the `includeWorkspaceFolder` optional argument.

https://github.com/microsoft/vscode/blob/1.58.2/src/vs/vscode.d.ts#L10546

Reply:
Done in db5b8077e4f4e909438c050c4b4b8813884fc6f9